### PR TITLE
installer: don't disable firewall

### PR DESCRIPTION
**Pull request by sielicki**

### CODE REVIEW

File: nix/installer.nix [hunk 1]
```diff
@@ -16,8 +16,8 @@
     [ "console=ttyS0,115200" ];
 
   documentation.enable = false;
-  # Not really needed. Saves a few bytes and the only service we are running is sshd, which we want to be reachable.
-  networking.firewall.enable = false;
+  networking.firewall.enable = true;
+  services.openssh.openFirewall = lib.mkForce true;
```
Comment (Mic92) [inline nix/installer.nix:20]: What do we need this firewall for?

Comment (sielicki) [reply nix/installer.nix:20]: If someone pulls this module into their config and they have something listening, why expose that unnecessarily? This is predominantly used on public clouds.

what do we need to disable it for?

Comment (Mic92) [reply nix/installer.nix:20]: This is an installer image, so people are unlikely to install services like databases that need a firewall. If they install network services they probably want to expose it to the network. Having no firewall reduces complexity and saves us some storage for not having to ship firewall binaries, which helps with using the kexec image on low-memory devices.

Comment (sielicki) [reply nix/installer.nix:20]: the background for this was to make this module less opinionated for usage in nixos-generators -- nearly all of those images formatconfigs are capable of defining their image type without imposing so much on the user like disabling their firewall from the defaults and/or building ZFS support, but I understand where you're coming from. Thanks.

Comment (Mic92) [reply nix/installer.nix:20]: Less opinionated should be probably part of nixpkgs, where we also have a installer profile. The primary use case of these images here are for use in nixos-anywhere in combination with disko.

**Pull request closed without merging**